projects/cdk/schematics/ng-update/v4/steps/constants/enums.ts

```diff
@@ -21,6 +21,27 @@ export const ENUMS_TO_REPLACE: ReplacementEnum[] = [
         },
         keepAsType: false,
     },
+    {
+        name: 'TuiCard',
+        replaceValues: {
+            TinkoffAirlines: 't-airlines',
+            TinkoffAirlinesCredit: 't-airlines-credit',
+            TinkoffBlack: 't-black',
+            TinkoffBusiness: 't-business',
+            TinkoffJunior: 't-junior',
+            TinkoffPlatinum: 't-platinum',
+        },
+        keepAsType: true,
+    },
+    {
+        name: 'TuiTimelineStatus',
+        replaceValues: {
+            Completed: 'completed',
```

Comment:
When we port this component to open source let's use more neutral `success | error | neutral`, like in status appearances.